react/features/mobile/callkit/middleware.js

```diff
@@ -268,7 +269,10 @@ function _onPerformSetMutedCallAction({ callUUID, muted: newValue }) {
         const { muted: oldValue } = getState()['features/base/media'].audio;
 
         if (oldValue !== newValue) {
-            dispatch(setAudioMuted(Boolean(newValue)));
+            const value = Boolean(newValue);
+
+            sendEvent(`callkit.audio.${value ? 'mute' : 'unmute'}`);
```

Comment:
We sometimes name events "xxx.audio.muted" (for example toolbar.audio.muted) and sometimes "xxx.audio.mute" like in the above. Maybe we should try to be consistent with the new events at least.

Reply:
Agreed!